kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -326,7 +154,10 @@ def concatenate(arrays):
 backend.cdgmm = cdgmm
 backend.modulus = Modulus()
 backend.subsample_fourier = SubsampleFourier()
-backend.fft = fft
+backend.fft = FFT(lambda x:torch.fft(x, 2, normalized=False),
+                  lambda x:torch.ifft(x, 2, normalized=False),
+                  lambda x:torch.irfft(x, 2, normalized=False, onesided=False),
+                  sanity_check)
```

Comment:
In the other backend you pass `lambda x: None`. So which one is it?

Reply:
i just didn't write any tests